pkg/agents/team2/message.go

```diff
@@ -0,0 +1,29 @@
+package team2
+
+import "github.com/SOMAS2021/SOMAS2021/pkg/messages"
+
+func (a *CustomAgent2) AskNeighbourHP() {
+	msg := messages.NewAskHPMessage(a.ID(), a.Floor(), a.Floor()+1)
+	a.SendMessage(msg)
+}
+
+func (a *CustomAgent2) CheckNeighbourHP() {
+	msg := a.ReceiveMessage()
+	for msg != nil {
+		if msg.MessageType() == 7 {
```

Comment:
This is not safe - you should use `messages.MESSAGE_TYPE_HERE` instead. The numbers may change (e..g one of the types was removed overnight)

Reply:
Example: https://github.com/SOMAS2021/SOMAS2021/blob/main/pkg/agents/team4/finalAgent/messages.go#L15

Reply:
Will fix shortly.

Reply:
Fixed.